src/devices/Ads1115/Ads1115.sln

```diff
@@ -0,0 +1,65 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29409.12
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Ads1115", "Ads1115.csproj", "{C60BB2A9-E067-4A0D-AF04-5D8B2FF1CE08}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Ads1115.Samples", "samples\Ads1115.Samples.csproj", "{71CE4785-8635-4100-B029-292A835FC852}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Device.Gpio", "..\..\System.Device.Gpio\System.Device.Gpio.csproj", "{E741DF97-C1D8-4322-AD63-88569D370D6A}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Linux-Debug|Any CPU = Linux-Debug|Any CPU
+		Linux-Release|Any CPU = Linux-Release|Any CPU
+		Release|Any CPU = Release|Any CPU
+		Windows-Debug|Any CPU = Windows-Debug|Any CPU
+		Windows-Release|Any CPU = Windows-Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{C60BB2A9-E067-4A0D-AF04-5D8B2FF1CE08}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{C60BB2A9-E067-4A0D-AF04-5D8B2FF1CE08}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{C60BB2A9-E067-4A0D-AF04-5D8B2FF1CE08}.Linux-Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Can we either remove the Solution or update it now that you have removed the configurations on your project?

Reply:
Sure, I'll clean that up or remove it. We should fix that latest after getting rid of those extra configs.

Reply:
I've deleted it for now. We should IMHO re-create these in a batch for all bindings when we've cleaned up the platform dependency.